kclvm/compiler_base/error/src/diagnostic/style.rs

```diff
@@ -3,36 +3,45 @@ use rustc_errors::Style;
 use termcolor::{Color, ColorSpec};
 
 /// 'DiagnosticStyle' defines all the styles that needed when displaying diagnostic message.
-#[derive(Copy, Clone, Debug, PartialEq)]
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
 pub enum DiagnosticStyle {
     Logo,
     NeedFix,
     NeedAttention,
     Helpful,
     Important,
     Url,
-    NoStyle,
```

Comment:
Why delete `NoStyle`?

Reply:
```
struct StyledChar<T>
where
    T: Clone + PartialEq + Eq + Style,
{
    chr: char,
    style: Option<T>,
}
```
`style` in `StyledChar` is optional, So I replaced `NoStyle` by `None`.